src/components/my-widgets-scores.jsx

```diff
@@ -61,7 +61,9 @@ const MyWidgetsScores = ({inst, beardMode}) => {
 	const handleShowOlderClick = () => setState({...state, isShowingAll: !state.isShowingAll})
 
 	let contentRender = <LoadingIcon />
-	if (isFetched) {
+	if (is_student) {
+		contentRender = <p>Students cannot view scores.</p>
```

Comment:
Could we modify this so that students can only view scores if the widget is in guest mode? Historically, we've been fine with student users being able to view score results, just anonymized ones. An even better solution would be to potentially filter out score results except for "All Guests", in edge cases where a student is given access to a widget that previously had non-anonymized scores.